server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -6769,6 +6771,7 @@ public boolean deleteNetworkOffering(final DeleteNetworkOfferingCmd cmd) {
 
         annotationDao.removeByEntityType(AnnotationService.EntityType.NETWORK_OFFERING.name(), offering.getUuid());
         if (_networkOfferingDao.remove(offeringId)) {
+            networkOfferingDetailsDao.removeDetails(offeringId);
```

Comment:
shouldn´t we first remove the details and than the oddering?

Reply:
thanks Daan.
fixed